### PR TITLE
Update documentation in apk.py

### PR DESCRIPTION
**Pull request by navisk13**
Update documentation for the apk module

##### SUMMARY

Fixes issue #9017 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

apk

##### ADDITIONAL INFORMATION

The `apk` module has a `world` parameter where you can specify a path to a custom world file. However, the documentation does not mention that the file is used only under certain conditions. I have added those details to the documentation, under the description for the `world` parameter.

```paste below

```

### CODE REVIEW

File: plugins/modules/apk.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
     default: false
   world:
     description:
-      - Use a custom world file when checking for explicitly installed packages.
+      - Use a custom world file when checking for explicitly installed packages. The file is used only when a value is provided for O(name), and O(state) is set to V(present) or V(latest).
```
Comment (felixfontein) [inline plugins/modules/apk.py:76]:
```suggestion
      - Use a custom world file when checking for explicitly installed packages.
        The file is used only when a value is provided for O(name), and O(state) is set to V(present) or V(latest).
```

**Comment by felixfontein** [comment]
Thanks for your contribution! Adding a linebreak should fix the problem with the too long line.

**Comment by navisk13** [comment]
Thank you @felixfontein!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@navisk13 thanks!

**Pull request merged**